examples/chef/devices/rootnode_contactsensor_lFAGG1bfRO.matter

```diff
@@ -29,11 +29,12 @@ server cluster Identify = 3 {
   readonly attribute enum8 identifyType = 1;
   readonly attribute command_id generatedCommandList[] = 65528;
   readonly attribute command_id acceptedCommandList[] = 65529;
-  readonly attribute event_id eventList[] = 65530;
+  readonly attribute optional event_id eventList[] = 65530;
```

Comment:
Optional versus not yet available is different. EventList is not optional, it's mandatory, but provisional.

Suggest "provisional" to be added as well, so that you have the distinction.

Reply:
I can add `provisional` flags however I don't think that is in the XML yet. Overall even if this becomes "official" I believe for backwards compat we will have to accept this missing (i.e. it will behave as optional).

I would prefer to fix this as a followup once we start accepting event list or we figure out how we handle provisional bits in general.

Reply:
Global attributes says `<attribute side="server" code="0xFFFA" define="EVENT_LIST"              type="array" entryType="event_id">EventList</attribute>`

So if we want provisional we have to update xml, update zap to understand the new xml and propagate it as data, change IDL to flag it and handle our parser and potential other logic. Seems like a separate followup or issue is a better course here (if we decide to even handle it ... for now our code does not seem to need it, may in the future)